source/main/assets/tutorial/fr/zettelkasten.md

```diff
@@ -0,0 +1,58 @@
+---
+title: "Working with Zettelkästen"
+author:
+  - The Zettlr Team
+  - Framatophe (trad.)
+date: 2020-07-04
+keywords:
+  - Zettelkasten
+  - Zettel
+  - Knowledge Management
+  - Niklas Luhmann
+...
+
+
+# Les Zettelkästen avec Zettlr 🗂
+
+Re-bonjour ! Vous venez de cliquer sur ce que nous appelons un « lien interne »  cela ressemble beaucoup à des liens wiki mais en fait c'est une fonctionnalité qui aide Zettlr à reconnaître comment vous organisez vos connaissances. Voyons d'abord ce qui vient de se passer après que vous ayez cliqué sur ce lien.
+
+Chaque fois que vous cliquez sur un lien interne, Zettlr fait deux choses : il lance une recherche globale sur le contenu du lien et essaie d'ouvrir le fichier correspondant. Comme vous pouvez le voir, le fichier contenant le tutoriel sur les références a disparu de la liste des fichiers sur la gauche, le champ de recherche globale contient maintenant le mot « zettelkasten » ­— et, surtout, vous êtes en train de regarder ce fichier !
```

Comment:
I remember that CodeMirror showed me a red dot behind « zettelkasten » , which is an indicator that accidentally a non-printable character has managed to sneak its way in there — it would be great if you could have a look at this! :)

Reply:
ok.